src/navigation/sectionalTabs/sectionalTabs.jsx

```diff
@@ -224,8 +224,7 @@ class SectionalTabs extends Component {
                 }),
                 ...(didPropsItemsChange && {
                     items,
-                    // let's select the first tab when there's a different items source
-                    selectedTabKey: get(items, '[0].key'),
+                    selectedTabKey,
```

Comment:
@morethanfire I see what you did in here, let me link this branch with the client so I can see how it behaves with the current Navigation progress, I'll get you updated hopefully tomorrow at morning at the scrum session.

Reply:
I already tested this changed, everything work fine in the side client with the current Navigation Changes, thanks @morethanfire

Reply:
Nice!